optimum/exporters/onnx/convert.py

```diff
@@ -645,6 +665,3 @@ def export(
         raise RuntimeError(
             "You either provided a PyTorch model with only TensorFlow installed, or a TensorFlow model with only PyTorch installed."
         )
-
-    config.fix_dynamic_axes(output)
```

Comment:
Why removing this? It doesn't work?

Reply:
It is a mistake, thanks for spotting!

Reply:
Okay, so we need to keep the definition of `DynamicAxisNameError` above and `return export_output` here also.